test/browser/main.fmf

```diff
@@ -4,8 +4,6 @@ require:
   - cockpit-ws
   - cockpit-system
   - criu
-  # HACK: https://bugzilla.redhat.com/show_bug.cgi?id=2269485
-  - slirp4netns
```

Comment:
This still worked in c10s, but broke rhel-10 gating tests. So after this lands, I'll do a new upstream release.